From 31515b13b469a8060b34901357fae1077f42dd2e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20Br=C3=BCns?= <stefan.bruens@rwth-aachen.de>
Date: Thu, 20 Jun 2019 03:31:23 +0200
Subject: [PATCH 04/14] Fix usage of 0 for null pointer constants

Summary: Depends on D21972

Reviewers: lukas, jgrulich, ngraham

Reviewed By: ngraham

Differential Revision: https://phabricator.kde.org/D21973
---
 src/declarative/pkupdates.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/declarative/pkupdates.cpp b/src/declarative/pkupdates.cpp
index 9fdb538..0832551 100644
--- a/src/declarative/pkupdates.cpp
+++ b/src/declarative/pkupdates.cpp
@@ -395,7 +395,7 @@ void PkUpdates::onFinished(PackageKit::Transaction::Exit status, uint runtime)
             if (upCount > 0) {
                 KNotification::event(KNotification::Notification, i18n("Software Updates Available"),
                                      i18np("You have 1 new update", "You have %1 new updates", upCount),
-                                     s_pkUpdatesIconName, 0, KNotification::Persistent);
+                                     s_pkUpdatesIconName, nullptr, KNotification::Persistent);
             }
         } else {
             qCDebug(PLASMA_PK_UPDATES) << "Check updates transaction didn't finish successfully";
@@ -418,7 +418,7 @@ void PkUpdates::onFinished(PackageKit::Transaction::Exit status, uint runtime)
             qCDebug(PLASMA_PK_UPDATES) << "Update packages transaction finished successfully";
             KNotification::event(KNotification::Notification, i18n("Updates Installed"),
                                  i18np("Successfully updated %1 package", "Successfully updated %1 packages", packages.count()),
-                                 s_pkUpdatesIconName, 0, KNotification::Persistent);
+                                 s_pkUpdatesIconName, nullptr, KNotification::Persistent);
             emit updatesInstalled();
         } else {
             qCDebug(PLASMA_PK_UPDATES) << "Update packages transaction didn't finish successfully";
@@ -445,7 +445,7 @@ void PkUpdates::onErrorCode(PackageKit::Transaction::Error error, const QString
         return;
 
     KNotification::event(KNotification::Error, i18n("Update Error"), details,
-                         s_pkUpdatesIconName, 0, KNotification::Persistent);
+                         s_pkUpdatesIconName, nullptr, KNotification::Persistent);
 }
 
 void PkUpdates::onRequireRestart(PackageKit::Transaction::Restart type, const QString &packageID)
-- 
2.43.0

